onnxruntime/core/common/cpuid_info.cc

```diff
@@ -54,7 +54,9 @@ CPUIDInfo::CPUIDInfo() noexcept {
       const int AVX_MASK = 0x6;
       const int AVX512_MASK = 0xE6;
       int value = XGETBV();
-      has_avx_ = (data[2] & (1 << 28)) && ((value & AVX_MASK) == AVX_MASK);
+      bool has_sse2 = (data[3] & (1 << 26));
+      bool has_ssse3 = (data[2] & (1 << 9));
+      has_avx_ = has_sse2 && has_ssse3 && (data[2] & (1 << 28)) && ((value & AVX_MASK) == AVX_MASK);
```

Comment:
Is the original code (i.e. ```(data[2] & (1 << 28)) && ((value & AVX_MASK) == AVX_MASK);```) not sufficient? Why do we need two more checks, ```has_sse2``` and ```has_ssse3```?